index.html

```diff
@@ -30,7 +30,8 @@
     </div>
 
     <footer>
-      <a href="https://github.com/greenpeace/planet4-test-swarm" target="_blank">source code</a>
+      <a href="https://github.com/greenpeace/planet4-test-swarm" target="_blank">source code</a> | 
+      <a href="https://github.com/greenpeace/planet4-test-swarm/tree/master/static">browser script to navigate from Jira</a>
```

Comment:
Shouldn't the script below moved inside the `static` folder? The link could point to the script directly.

Maybe limit the text to just "Jira script", to keep it short.